instructor/common.py

```diff
@@ -274,7 +282,9 @@ def assignments_variables(request):
     form = AssignmentForm(instance=assignment)
     return render_to_response('instructor/assignment_select_variables.html',
                               {'form': form,
-                               'assignment': assignment
+                               'treatments_created': treatments_created,
+                               'assignment': assignment,
+                               'assignment_name': assignment.name
```

Comment:
It looks like `assignment` and `assignment_name` are unused in `assignment_select_variables.html`, can you remove these lines if it's not necessary?